node/src/blockchain/blockchain_bridge.rs

```diff
@@ -270,13 +270,25 @@ impl BlockchainBridge {
     }
 
     fn handle_retrieve_transactions(&mut self, msg: &RetrieveTransactions) -> Result<(), String> {
+        let end_block = match self.blockchain_interface.get_block_number() {
+            Ok(eb) => BlockNumber::Number(eb),
+            Err(e) => {
+                warning!(
+                    self.logger,
+                    "Using 'latest' block number instead of a literal number. {:?}",
```

Comment:
Maybe it should be informational instead of a warning? It's not meant to sound bad or dangerous; only that we couldn't fetch the current block number and so now we're just going to query using BlockNumber::Latest.

Reply:
Yeah, it sounds like something that ought to be an `info!()` instead of a `warning!()`. I believe the way we've got it set up now, all warning and error logs are broadcast to the UIs. (I'm not 100% sure of that; @bertllll can correct me if I'm wrong.)